go/store/prolly/map.go

```diff
@@ -124,10 +124,19 @@ func DiffMaps(ctx context.Context, from, to Map, cb DiffFn) error {
 	return diffOrderedTrees(ctx, from.tuples, to.tuples, cb)
 }
 
+// RangeDiffMaps returns diffs within a Range. See Range for which diffs are
+// returned.
 func RangeDiffMaps(ctx context.Context, from, to Map, rng Range, cb DiffFn) error {
 	return rangeDiffOrderedTrees(ctx, from.tuples, to.tuples, rng, cb)
 }
```

Comment:
Do we have a use case for this old function, given the new one?

Reply:
I believe we do. We could implement range diffs in the sql layer for our diff tables instead of applying a filter.